### ISSUE TITLE
The license

### ISSUE THREAD

**Issue by snowdeer** [hediet/browser-ext-github-monaco#48]
Thank you for implementing nice codes for this one 😃👍

There is no license file in this project. 
Would you be willing to add an OSI-approved license like 
the MIT, the Apache, or the GPL License to the project ?

**Comment by hediet** [comment]
You are right. I think I once had commercial interests with this extension, but that's off the table now.
I'm fine with MIT (also needs to be changed in package.json).

What are your plans with this code?

**Comment by snowdeer** [comment]
Thank you for your answer and MIT License. 😃  
I think this is a very interesting feature, but it does not support GitHub Enterprise yet.
So I want to customize it for our company's GitHub premise version.
It will be developed by just myself personally, and it will be not commercial.
Thank you !! 😄
